### PR TITLE
INT-2767: Prepare for TinyMCE 6; change license to MIT

### PR DESCRIPTION
**Pull request by tiny-james**

### CODE REVIEW

File: src/main/ts/component/Editor.ts [hunk 1]
```diff
@@ -105,17 +109,21 @@ class TinyMceEditor extends HTMLElement {
     });
   };
 
-  private _formDataHandler = (evt: Event) => {
+  private _formDataHandler = (evt: FormDataEvent): void => {
     const name = this.name;
-    if (name !== null) {
-      const data = (evt as any).formData as FormData;
-      data.append(name, this.value);
+    if (name != null) {
```
Comment (techtangents) [inline src/main/ts/component/Editor.ts:114]: Did you mean `!==` here?

Comment (tiny-james) [reply src/main/ts/component/Editor.ts:114]: It was deliberate. We have a lint to check for `!=` for anything that is not null.

In this case while I don't expect `undefined` I did want to handle it.

Also I'm pretty sure you work somewhere else now! :-P

Comment (techtangents) [reply src/main/ts/component/Editor.ts:114]: :)

Comment (EkimChau) [reply src/main/ts/component/Editor.ts:114]: There was a strange in-equality in the force that rippled through the fabric of git, that drew him to this PR

**Review by EkimChau — APPROVED** [review]

File: src/main/ts/component/Editor.ts (hunk 1 above)
Comment (techtangents) [reply src/main/ts/component/Editor.ts:114]: I discovered the other day that my github account was using my Tiny email. After fixing that I've been getting lots of notifications :)

I only commented to say hi :)

**Review by jscasca — APPROVED** [review]

**Pull request merged**